bcsv.py

```diff
@@ -8,7 +8,7 @@
 
 def printHex(s):
     for i in range(0, len(s), 4):
-        print ''.join(['%02X'%ord(x) for x in s[i:min(i+4, len(s))]]),
+        print(''.join(['%02X'%ord(x) for x in s[i:min(i+4, len(s))]]),)
```

Comment:
These old-style print statements mean that the code is in Python 2, and updating them implies that you've updated the code to run in Python 3. Have you actually verified that they run in Python 3? There are likely other differences, like file and string APIs.

Reply:
Also, there are already existing tools that will do a Python 2 to Python 3 conversion automatically, like 2to3.